packages/host/app/components/matrix/forgot-password.gts

```diff
@@ -437,4 +443,17 @@ export default class ForgotPassword extends Component<Signature> {
       throw e;
     }
   });
+
+  @action
+  private backToLogin() {
+    let cardController = getOwner(this)!.lookup('controller:card') as any;
+    if (!cardController) {
+      throw new Error(
+        'OperatorModeStateService must be used in the context of a CardController',
+      );
+    }
+    cardController.sid = null;
+    cardController.clientSecret = null;
+    this.args.onLogin();
```

Comment:
I'm confused here–reading `backToLogin` and then there's a call to `onLogin` sounds odd. Maybe you can make a `nullifyPasswordResetQueryParams` method and call from it where you toggle the form to login.